content/node-runners/node-migration.md

```diff
@@ -41,6 +41,45 @@ And locating the `Binds`:
 If you cannot find it or you didn't create a volume for the data dir, you will have to [exec](https://docs.docker.com/engine/reference/commandline/exec/) in to the
 docker container and follow our linux/raspberry instructions on how to retrieve it.
 
+## Finding `data-dir` in dockerized node (MacOS host)
+
+When using Docker Desktop for MacOS, the containers and all data are stored within the Linux VM. Since the new version of Docker (mine is 20.10.7) uses socket instead of TTY to communicate with the VM, we will use the NC command and open the debug shell socket to interact with the virtual machine:
+
+```bash
+nc -U ~/Library/Containers/com.docker.docker/Data/debug-shell.sock
+```
+
+Find your data directory (keystore, testnet2 folders & nodeui-pass file):
+
+```bash
+/ # ^[[34;5Rls /var/lib/docker/volumes/myst-data/_data/
```

Comment:
`^[[34;5R` Whats that supposed to be?

Reply:
It's a prefix assigned by default when opening a shell. I think we can remove it.

Reply:
We should